package.json

```diff
@@ -111,7 +111,8 @@
     "start": "concurrently --names server,frontend \"npm run start:server\" \"cross-env GATSBY_BASE_URL=http://localhost:8080 gatsby develop --port 3000\"",
     "e2e": "start-server-and-test start http://localhost:3000 test:e2e",
     "e2e-on-build": "cross-env CYPRESS_baseUrl=http://localhost:8080 start-server-and-test start:server:e2e-on-build http://localhost:8080 test:e2e",
-    "badge": "cross-env NODE_CONFIG_ENV=test TRACE_SERVICES=true node scripts/badge-cli.js"
+    "badge": "cross-env NODE_CONFIG_ENV=test TRACE_SERVICES=true node scripts/badge-cli.js",
+    "build-docs": "rimraf api-docs/ && jsdoc --pedantic -c ./jsdoc.json . && mkdir -p ./api-docs/frontend/images/ && cp ./frontend/images/logo.svg ./api-docs/frontend/images/logo.svg"
```

Comment:
I wonder if this needs `mkdirp` to be cross-platform.

Reply:
I think we will need to use `mkdirp` too 👍

Reply:
Ah yes I forgot to remove the mkdir/cp commands when I changed the image to an absolute URL. Have just removed them.